docusaurus/docs/reference/backup/10-controller.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Controller Backup
+sidebar_label: Controller
+---
+
+## Backup
+
+The best way to ensure continuity is to run multiple controllers. You can recover from the catastrophic loss of all
+controllers in the cluster by creating a new single controller which can become the inaugural leader of a new cluster.
+
+The three stateful aspects of the controller are the configuration YAML, PKI, and the
+database. Ensure you back up all the files mentioned in the controller's configuration YAML file (keys, certs, root CA
+bundle, and the database), and the config YAML file itself.
+
+If you're using the Linux service you may accomplish this by triggering the database snapshot operation and backing up
+the `/var/lib/ziti-controller` directory.
+
+### Backing up the Configuration YAML
```

Comment:
This all feels like "guide related" and not reference. A reference doc shouldn't talk about how to back something up, it should just tell you "this file is important and it's located here and it has this purpose"... This content feels more like it belongs in the guide section and is probably bespoke per install (linux/docker/kubernetes)

Reply:
We decided to move it from guides to reference.